src/neuroconv/datainterfaces/ophys/basesegmentationextractorinterface.py

```diff
@@ -89,23 +110,24 @@ def get_metadata_schema(self) -> dict:
         fill_defaults(metadata_schema, self.get_metadata())
         return metadata_schema
 
-    def get_metadata(self) -> dict:
+    def get_metadata(self) -> dict:  # noqa: D102
         from ...tools.roiextractors import get_nwb_segmentation_metadata
 
         metadata = super().get_metadata()
         metadata.update(get_nwb_segmentation_metadata(self.segmentation_extractor))
         return metadata
 
-    def get_original_timestamps(self) -> np.ndarray:
+    def get_original_timestamps(self) -> np.ndarray:  # noqa: D102
         reinitialized_extractor = self.get_extractor()(**self.source_data)
         return reinitialized_extractor.frame_to_time(frames=np.arange(stop=reinitialized_extractor.get_num_frames()))
 
-    def get_timestamps(self) -> np.ndarray:
+    def get_timestamps(self) -> np.ndarray:  # noqa: D102
         return self.segmentation_extractor.frame_to_time(
             frames=np.arange(stop=self.segmentation_extractor.get_num_frames())
         )
 
     def set_aligned_timestamps(self, aligned_timestamps: np.ndarray):
+        """set the aligned timestamps for the segmentation extractor."""
```

Comment:
Again here, needs Parameters section for aligned_timestamps

Reply:
Changed to noqa, this docstring is identical and should be inherited from the base class:

https://github.com/catalystneuro/neuroconv/blob/0d5d6ccb430e545b4b899e861876d600e04c5fbf/src/neuroconv/basetemporalalignmentinterface.py#L42-L56